tests/impl/model/sqlalchemy_model/postgresql/test_array_datatype.py

```diff
@@ -1,39 +1,50 @@
 # Standard Library Imports
 from datetime import datetime
 
+from sqlalchemy import types as sa_types
+
 # Protean
 import pytest
 
 from protean.core.aggregate import BaseAggregate
 from protean.core.exceptions import ValidationError
 from protean.core.field.basic import (
-    Array,
     Auto,
     Boolean,
     Date,
     DateTime,
     Float,
     Identifier,
     Integer,
+    List,
     String,
 )
 
 
 class ArrayUser(BaseAggregate):
     email = String(max_length=255, required=True, unique=True)
-    roles = Array()  # Defaulted to String Content Type
+    roles = List()  # Defaulted to String Content Type
 
 
 class IntegerArrayUser(BaseAggregate):
     email = String(max_length=255, required=True, unique=True)
-    roles = Array(content_type=Integer)
+    roles = List(content_type=Integer)
```

Comment:
If I use `List(content_type=Integer)` in a sqlite db, will it work?

Reply:
Yep, it works 👍

Reply:
Tests are here: https://github.com/proteanhq/protean/commit/50ce8383970196adfe398d8b1b2d366c230de299

I will fix defaulting as part of https://github.com/proteanhq/protean/issues/335